src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -195,10 +195,10 @@ def list_jobs(self):
 
             if not len(status_set):
                 return "No running jobs!"
-            status_set = sorted(status_set, key=lambda s: ObjectId(s['_id']).generation_time.timestamp())
+            status_set = sorted(status_set, key=lambda s: ObjectId(s['job_id']).generation_time.timestamp())
 
             for status in status_set:
-                status['creation_time'] = datetime.datetime.strftime(ObjectId(status_set[0]['_id']).generation_time, "%Y-%m-%d %H:%M:%S")
+                status['creation_time'] = datetime.datetime.strftime(ObjectId(status['job_id']).generation_time, "%Y-%m-%d %H:%M:%S")
```

Comment:
Same here? we can just use 'created' instead of 'creation_time', I think?

Reply:
💯